randovania/generator/pickup_pool/pickup_creator.py

```diff
@@ -137,17 +137,19 @@ def create_nothing_pickup(resource_database: ResourceDatabase, model_name: str =
     )
 
 
-def create_visual_etm() -> PickupEntry:
+def create_visual_etm(game: RandovaniaGame = RandovaniaGame.METROID_PRIME_ECHOES,
+                      model_name: str = echoes_items.USELESS_PICKUP_MODEL,
+                      pickup_name: str = "Unknown item") -> PickupEntry:
```

Comment:
Don't know why I look at this in a 5 minute break...
Why using the defaults?
Afaik there should be a test for am2r using this function, which now falls back to the defaults and would break if the test contains an unknown item (which it maybe should).

Reply:
> So that all the other games using this method don't suddenly have their behaviour changed.

But the default is just wrong for other games and if you already touch it, I prefer a proper refactor.